studiopress/genesis-responsive-slider
# Deploy only 'Tested up to' change to `svn`

* Thanks to Phil's idea, when a branch starts with `bump-tested-up-to`, this will deploy to SVN a bump of the 'Tested up to' version in `trunk/readme.txt` and `tags/<version>/readme.txt`, and nothing else
* There's a [manual approval](https://github.com/studiopress/genesis-responsive-slider/pull/23#issuecomment-1101855184) required to deploy
* This PR isn't deploying yet, it intentionally doesn't have `svn ci` or the context with the password




### How to test

1. Look at https://github.com/studiopress/genesis-responsive-slider/pull/25
2. Expected: The CircleCI job `svn-deploy-bump-tested-up-to` does not run on the latest commit
3. Look at https://github.com/studiopress/genesis-responsive-slider/pull/24
4. [Go to](https://app.circleci.com/pipelines/github/studiopress/genesis-responsive-slider/63/workflows/e8638f27-a9e1-4368-885c-7a5e5fc0ff18) approve the job: <img width="1046" alt="approve-job" src="https://user-images.githubusercontent.com/4063887/163893777-e2f573e1-2014-4187-a435-f2fb037c4099.png"> (This might actually expire if we don't approve it in a certain time. Also, it won't actually deploy, as it doesn't have `svn ci`)
5. Expected: The job `svn-deploy-bump-tested-up-to` runs, and the `svn diff` at the end of the job looks right: <img width="780" alt="Screen Shot 2022-04-18 at 6 46 04 PM" src="https://user-images.githubusercontent.com/4063887/163893990-4966daa6-1e51-425f-a8fa-53be7c591640.png">

### Documentation
No documentation required. 



### Suggested changelog entry

Deploy only 'Tested up to' change to `svn`
